packages/manager/modules/anthos/src/dashboard/ips/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,22 @@
+{
+  "anthos_dashboard_ips_title": "Addresses IP",
```

Comment:
```suggestion
  "anthos_dashboard_ips_title": "Addresses IPs",
```
Not sure though

Reply:
Yes better, in others content they use IPs.
So for consistency i will use IPs.

Thanks